components/src/store/actions/form.ts

```diff
@@ -0,0 +1,15 @@
+import { TypeKeys } from "./index";
+
+export interface AddSubmittedFormId {
+    type: TypeKeys.ADD_SUBMITTED_FORM_ID;
+    key: string;
+}
+
+// Add a HubSpot formId to submitted forms list.
+export const addSubmittedFormId = (formId: string) => (dispatch, _getState) => {
+    const action: AddSubmittedFormId = {
```

Comment:
Or `SubmitHubSpotForm`, maybe? 🤔  We might want to think a bit more on this one and maybe move some logic into the action if it made sense.